### PR TITLE
Support for %hostname%, %uuid% and %serial% to eventDetailURLForEvent

### PR DESCRIPTION
**Pull request by hughneale**
Hello,

I've added support for `%hostname%`, `%uuid%` and `%serial%` to eventDetailURLForEvent to provide additional system information for blocked events. We're using the `%machine_id%` as our own unique identifier and it'd be useful to be able to provide some additional system information in event detail URLs for external systems that don't have access to the machine_id. 

Thanks

Best,

H

### CODE REVIEW

**Comment by hughneale** [comment]
@googlebot I signed it!

File: Source/common/SNTBlockMessage.m [hunk 1]
```diff
@@ -122,6 +126,18 @@ + (NSURL *)eventDetailURLForEvent:(SNTStoredEvent *)event {
     formatStr = [formatStr stringByReplacingOccurrencesOfString:@"%machine_id%"
                                                      withString:config.machineID];
   }
+  if (hostname) {
```
Comment (russellhancox) [inline Source/common/SNTBlockMessage.m:129]: Use `hostname.length` here - that will ensure the value is both non-nil and not zero-length. Same for uuid and serial below

Comment (hughneale) [reply Source/common/SNTBlockMessage.m]: Thanks, I've amended this. 👍

**Comment by hughneale** [comment]
Thank you!

**Comment by russellhancox** [comment]
Thank you for the contribution! They'll be included in our next release, which will likely be in a few weeks as we're trying to get another long-awaited feature implemented.

**Pull request merged**